src/ZombieDriver.php

```diff
@@ -94,13 +94,14 @@ public function stop()
     public function reset()
     {
         $js = <<<JS
-browser.destroy();
-browser = null;
```

Comment:
I'm not having problems with that exactly. I've just imagined, based on what I've found in the code (no window switching handling) that this might be a problem and reported it in https://github.com/Behat/MinkExtension/issues/220 . And in there you told that window switching isn't handled in any way on session reset as well.

So I wrote the test for Zombie & Selenium2 drivers (only they support window manipulations) to prove that. I've based my test on the fact, that last opened window can be accessed after `reset` method call, but that only worked for Selenium, but not Zombie. Hence the problem I'm facing.